### DISCUSSION TITLE
整合包多样化

### DISCUSSION THREAD

**Discussion by mengd143** [Meloong-Git/PCL#1117] [提问] [answered]
就比如Xplus，人家可以支持fabric，你上面就只有forge的版本

**Comment by Big-Cake-jpg** [comment, marked as answer]
彻头彻尾的与 PCL2 无关。

只有 Forge 版本是因为作者在 CurseForge 上只发布了 Forge 版本，你就算用 HMCL 或者 BakaXL 去搜也是一样找不到 Fabric 版本的。难不成你想让龙猫强行加一个不存在的版本进去啊。

这边建议出门左转 beta.mcbbs.net

![image](https://user-images.githubusercontent.com/76810494/216228305-4b15f742-a06c-4e6a-9ee1-76244b11d7fc.png)
